gobblin-modules/gobblin-azkaban/src/main/java/org/apache/gobblin/service/modules/orchestration/AzkabanSpecProducer.java

```diff
@@ -138,8 +139,12 @@ public void close() throws IOException {
     return new CompletedFuture<>(_config, null);
   }
 
-  @Override
   public Future<?> deleteSpec(URI deletedSpecURI) {
```

Comment:
Why is this still here when it has been removed from the interface? If this API is required then add one with the default implementation to the interface.